client/Makefile

```diff
@@ -13,7 +13,7 @@
 #
 
 FN_VER_MAJOR	=	2
-FN_VER_MINOR	=	9.1
+FN_VER_MINOR	=	9.2
```

Comment:
Let's make this version 2.10.1

Reply:
Oh I guess I should have asked for 2.10.0.  Oh well, we'll skip the .0 this time.